frontend/ui/size.lua

```diff
@@ -77,6 +89,11 @@ local Size = {
         vertical_default = Screen:scaleBySize(2),
         vertical_large = Screen:scaleBySize(5),
     },
+    maxItemsNorm = {
```

Comment:
But this one should be snake_case, like its siblings.
Also, no real need to abbreviate, especially this "Norm". What does it mean? If it's normalized, I guess all the things in this size.lua are normalized values :)

Reply:
Normalized against physical size.
So any screen size will show this item with the same physical size.

Reply:
I agree, the norm seems redundant.